src/com/konovalova/lesson16/GenericClass.java

```diff
@@ -0,0 +1,35 @@
+package com.konovalova.lesson16;
+
+import java.util.Scanner;
+
+public class GenericClass {
+
+    public static <T> void fill(T [] myArray) {
+        T [] arr = myArray;
```

Comment:
why don't you like "myArray" ? You know that you creating one more reference to the same array object? :)